src/react/components/connect.tsx

```diff
@@ -33,6 +33,7 @@ export class Connect extends React.Component<any> {
     }
   };
   render() {
+    //@ts-ignore
```

Comment:
Why do we need this?

Reply:
tsc shows me an error:
Cannot invoke an expression whose type lacks a call signature. Type 'ReactNode' has no compatible call signatures.